app/parseBoard.py

```diff
@@ -106,14 +107,52 @@ def bullyPathing(converted_data, pathBoard):
                     y = a['y']
                     if(x < converted_data['board']['width'] and x > 0):
                         if(pathBoard[y][x+1] != -1):
-                            board[y][x+1] = 3
+                            board[y][x+1] = SMALLER_SNAKE_FUTURE_HEAD
                         if(pathBoard[y][x-1] != -1):
-                            board[y][x-1] = 3
+                            board[y][x-1] = SMALLER_SNAKE_FUTURE_HEAD
                     if(y < converted_data['board']['width'] and y > 0):
                         if(pathBoard[y+1][x] != -1):
-                            board[y+1][x] = 3
+                            board[y+1][x] = SMALLER_SNAKE_FUTURE_HEAD
                         if(pathBoard[y-1][x] != -1):
-                            board[y-1][x] = 3
+                            board[y-1][x] = SMALLER_SNAKE_FUTURE_HEAD
+    return board
+
+
+def cowardPathing(converted_data, pathBoard):
+    """
+    Finds Snakes that are LARGER than us and assigns the area around their head with the LARGER_Snake_Future_Head
+
+    May need logical update for snake bodies but it should be fine
+
+    Args:
+        converted_data (json): python readable json
+        path (list): path from a*
+
+    Return:
+        updated board (list) with potentially new values around LARGER snakes heads
+    """
+    me = converted_data["you"]["id"]
+    board = pathBoard
```

Comment:
Why are we dealing with board and pathBoard as two seperate reference variables to the same thing? If you meant to copy it. `Import copy`, `copy.deepcopy(list)`

Reply:
Do you have to use deepcopy to protect the original? Like is python actually doing a yeold pointer copy in Tates code? I thought that python was high enough level that it had abstracted away from the pointer copy unless you explicitly wanted to...

Reply:
Python just gives the reference. Any changes to one happens to the other

Reply:
So in any place where we pass a parameter and assign it locally, we should instead only use the parameter that was passed? This makes sense if python is pass by reference...

Reply:
Also means we can skip returning a bunch of stuff if this is true.